src/rust/madara/sources/worldmanhwas/res/source.json

```
@@ -15,4 +15,4 @@
 			"name": "Popular"
 		}
 	]
-}
+}
```

make sure there's a newline here
```suggestion
}

```